net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -391,148 +391,136 @@ mwan3_create_iface_iptables()
 {
 	local id family
 
-	config_get family $1 family ipv4
-	mwan3_get_iface_id id $1
+	config_get family "$1" family ipv4
+	mwan3_get_iface_id id "$1"
 
 	[ -n "$id" ] || return 0
 
-	if [ "$family" == "ipv4" ]; then
+	if [ "$family" = "ipv4" ]; then
 		$IPS -! create mwan3_connected list:set
 
 		if ! $IPT4 -S mwan3_ifaces_in &> /dev/null; then
 			$IPT4 -N mwan3_ifaces_in
 		fi
 
-		if ! $IPT4 -S mwan3_iface_in_$1 &> /dev/null; then
-			$IPT4 -N mwan3_iface_in_$1
+		if ! $IPT4 -S "mwan3_iface_in_$1" &> /dev/null; then
+			$IPT4 -N "mwan3_iface_in_$1"
 		fi
 
-		$IPT4 -F mwan3_iface_in_$1
-		$IPT4 -A mwan3_iface_in_$1 \
-			-i $2 \
+		$IPT4 -F "mwan3_iface_in_$1"
+		$IPT4 -A "mwan3_iface_in_$1" \
+			-i "$2" \
 			-m set --match-set mwan3_connected src \
 			-m mark --mark 0x0/$MMX_MASK \
```

Comment:
Why is $MMX_MASK quoted above but not here?

Reply:
I haven't quoted all the quotation that shellcheck recommends.
I didn't want to change too much.

Reply:
Okay - it probably makes sense to at least be consistent with what is quoted. Although, it probably doesn't matter, if somehow a space got into the MMX_MASK one of the previous iptables command would have failed.